### PR TITLE
Base/Doc Descriptions

### PR DESCRIPTION
**Pull request by TelegramSam**
Initial attempt at base/doc forms as discussed in working group.
For discussion in WG meeting

### CODE REVIEW

**Comment by decentralgabe** [comment]
Thanks, Sam!
I made a PR against your branch with some editorialization changes. https://github.com/TelegramSam/trust-establishment/pull/1

I will note that the data model section is not up to date with your change - but I think this is OK. Your change makes the optionality clear, and is useful to merge for a fruitful discussion on the next call. Then we can handle the rest of the spec.

**Review by decentralgabe — APPROVED** [review]
conditional approval upon the merge of https://github.com/TelegramSam/trust-establishment/pull/1

File: spec/spec.md [hunk 1]
```diff
@@ -90,6 +90,20 @@ Examples in this document use the Verifiable Credentials Data Model [[spec:VC-DA
 
 [[ref:Trust Establishment Documents]] are objects that articulate information a [[ref:Party]] publishes in order to share _assertions_ about a set of [[ref:DID]]-identified entities. [[ref:Issuers]], [[ref:Verifiers]], and [[ref:Holders]] may find utility in understanding a [[ref:Party]]'s assertions in making decisions in decentralized web interactions, some of which are covered in the [Usage](#usage) section.
 
+There is a base data model, along with document based adapatations that allow for aspect oriented uses.
```
Comment (mikekebert) [inline spec/spec.md:93]: adapatations -> adaptations

File: spec/spec.md [hunk 2]
```diff
@@ -116,12 +130,28 @@ Examples in this document use the Verifiable Credentials Data Model [[spec:VC-DA
...
+  "schema": "https://example.com/trusted-supplier.schema.json",
+  "entity": "did:example:alice",
+  "properties": {
+    "on_time_percentage": 92,
+    "goods": ["applewood", "hotel buffet style", "thick cut"]
+  }
+}
+```
+
+There are several document representations aligned around specific organizations of data: Topic Oriented, Entity Oriented, Entity Topic Oriented, and Set Oriented. Each representation is optomized around a particular indexing vector, but all contain the necessary information to produce the base data model for each assertion.
```
Comment (mikekebert) [inline spec/spec.md:147]: optomized -> optimized

File: spec/spec.md [hunk 3]
```diff
@@ -142,6 +172,104 @@ Examples in this document use the Verifiable Credentials Data Model [[spec:VC-DA
   }
 }
 ```
+::: Entity Oriented Example
+
+This pulls the topic to the top of the document, and indexes properties by entity.
```
Comment (mikekebert) [inline spec/spec.md:177]: Is this supposed to say "This pulls the entity to the top of the document and indexes properties by topic"?

File: spec/spec.md [hunk 4]
```diff
@@ -142,6 +172,104 @@ Examples in this document use the Verifiable Credentials Data Model [[spec:VC-DA
...
+      "https://example.com/other.schema.json": {
+        "foo": "baz"
+      }
+    }
+  }
+}
+```
+::: Set Oriented Example
+
+This document specifies neither entity or topic at the document level, and indexes by entity, then topic.
```
Comment (mikekebert) [inline spec/spec.md:231]: This doesn't seem to be indexed by entity, then topic. This seems to be just a collection of base model entries.

File: spec/spec.md [hunk 5]
```diff
@@ -142,6 +172,104 @@ Examples in this document use the Verifiable Credentials Data Model [[spec:VC-DA
...
+    {
+      "schema": "https://example.com/other.schema.json",
+      "entity": "did:example:carol",
+      "properties": {
+        "foo": "baz"
+      }
+    }
+  ]
+}
+```
```
Comment (mikekebert) [inline spec/spec.md:272]: Should we have an example where "This document specifies neither entity or topic at the document level, and indexes by topic, then entity."?

**Review by mikekebert — COMMENTED** [review]
I found two typos (not sure how careful we have to be with those right now).
I also had a few questions, hopefully not too pedantic.

**Pull request merged**